sklearn/cluster/_optics.py

```diff
@@ -143,6 +145,9 @@ class OPTICS(ClusterMixin, BaseEstimator):
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
+    heap : {'simple_heap','updateable_heap'}
+        Name of heap to be used.
```

Comment:
What are the pros and the cons of each heap? The user (and the PR reviewer :) should be given some guidance about how to choose the value of this parameter.